xml/admin_operating_services.xml

```diff
@@ -233,4 +233,127 @@ ceph-rbd-mirror@.service
 <screen>&prompt.smaster;salt-run state.orch ceph.restart.ganesha</screen>
   </sect2>
  </sect1>
+ <sect1 xml:id="ceph-cluster-shutdown">
+  <title>Shutdown and Start of the Whole &ceph; Cluster</title>
+
+  <para>
+   There are occasions when you need to stop all &ceph; related services in the
+   cluster in the recommended order, and then be able to simply start them
+   again. For example, in case of a planned power outage.
+  </para>
+
+  <procedure>
+   <title>Shutting Down the Whole &ceph; Cluster</title>
+   <step>
+    <para>
+     Shut down or disconnect any clients accessing the cluster.
```

Comment:
Do we want to provide CLI instruction?

Reply:
The problem here is that the clients will be significantly different and the steps will vary.  In the case of CephFS and Ganesha, unmount any filesystems from Ceph.  For iSCSI, the steps might involve disconnecting LUNs in a VMware GUI.  For RGW, any automation for S3 clients needs to be disabled.  

I don't think we will have generic instructions that would apply to a specific customer and I doubt my description would be that helpful.

Reply:
Agree